packages/contracts/src/plugins/governance/majority-voting/MajorityVotingBase.sol

```diff
@@ -6,7 +6,7 @@ import {ERC165Upgradeable} from "@openzeppelin/contracts-upgradeable/utils/intro
 import {Initializable} from "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
 import {SafeCastUpgradeable} from "@openzeppelin/contracts-upgradeable/utils/math/SafeCastUpgradeable.sol";
 
-import {ProposalUpgradeable, ProposalBase} from "../../../core/plugin/proposal/ProposalUpgradeable.sol";
+import {ProposalUpgradeable, IProposal} from "../../../core/plugin/proposal/ProposalUpgradeable.sol";
```

Comment:
we don't seem to be using `IProposal`. Your choice whether to remove it or leave it.

Reply:
Since it is a redundant import, I removed it in 772b4f448ae67dfd00a7b3ecaea7a7ae8223b369